lib/patrol.js

```diff
@@ -82,7 +71,7 @@ const patrol = function (stateDescriptor) {
       const fetcher = findClosestFetcher(dataStore, path);
 
       if (fetcher) {
-        fetcher(setCh, path, convertToNative(getStateByPath(currentState, path)));
+        fetcher(rootCursor.refine(path), rootCursor);
```

Comment:
This allows fetchers to have a consistent api with persisters.

The main usecase will only need the refined cursor and not the rootCursor, but sometimes you needs information thats in a parent cursor. For instance an id. give the 2 cursors and the fact that a cursor has a path member, you can use the rootCursor to find the piece of state you need in those rare cases.

I've not ever had a need for this in fetchers yet, but I have in persisters, and I wanted to keep that api consistent